ui-library/stories/Button.stories.ts

```
@@ -0,0 +1,53 @@
+    color: 'success',
+  },
+};
+
+export const Warning: Story = {
+  args: {
+    label: 'Button',
+    color: 'warning',
+  },
+};
+
+export const Error: Story = {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)